src/SFA.DAS.QnA.Database/projects/roatp/project.json

```diff
@@ -220,6 +220,12 @@
                "SequenceNo":7,
                "SectionNo":4,
                "name":"Quality and high standards in apprenticeship training"
+            },
+			{ 
```

Comment:
Can you fix the indenting, and the trailing space in the section name?

Reply:
I was using Notepad++, which seems that renders the indenting differently than VS. It’s fixed now.